pytorch_lightning/trainer/trainer.py

```diff
@@ -444,6 +444,12 @@ def __init__(
         self.signal_connector = SignalConnector(self)
         self.tuner = Tuner(self)
 
+        # UserWarning for max_epochs if not set
+
+        warnings.warn("max_epochs not set, defaulted to 1000 epochs until stop.") if self.max_epochs is None else print(
+            f"max epochs set to {self.max_epochs}"
+        )
```

Comment:
```suggestion
        if self.max_epochs is None and max_steps == -1 and max_time is None:
            warnings.warn("`max_epochs` is not set. By default it will run for 1000 epochs.")
        )
```
print statement is not required.